.github/workflows/docker-build-multiarch.yml

```diff
@@ -0,0 +1,248 @@
+---
+name: Build and push docker image
+
+# This workflow is based on, but diverges from the example here:
+# https://docs.docker.com/build/ci/github-actions/multi-platform/#distribute-build-across-multiple-runners
+
+on:
+  workflow_call:
+    inputs:
+      image-name:
+        description: image to build, including registry (only ghcr.io supported)
+        required: true
+        type: string
+      registry:
+        description: registry to push image to (only default ghcr.io is currently supported)
+        required: false
+        type: string
+        default: ghcr.io
```

Comment:
NIT: Odd that `image_name` requires the registry to be specified, but then you specify `registry` right after?

Maybe double-check this? I can imagine accidents if someone specifies an `image_name` registry and `registry` parameters that don't agree.

Reply:
Good catch. Sorting that out now.

Reply:
`image-name` no longer requires registry, but does require the username for namespacing within the registry (e.g. `rocker/r-ver` or `RMI-PACTA/workflow.pacta`)